### DISCUSSION TITLE
Bind Super Key

### DISCUSSION THREAD

**Discussion by stodd1031** [bbidulock/icewm#555] [Q&A] [answered]
Is it at all possible to bind only they super(windows) key to a command in the "keys" config file?
Something like this is what I'm looking for,
`key "Super"         firefox`
NOT like this
`key "Super+f"         firefox`

**Comment by r-a-cristian-93** [comment, marked as answer]
You can change it in the "keys" config file but the super keys are named "Super_L" and Super_R".
Obs: for me opens both the menu and firefox :))

`key "Super_L"  firefox`
